doc/tutorial/alphabet/index.md

```diff
@@ -0,0 +1,329 @@
+# Alphabets in SeqAn3 {#tutorial_alphabets}
+
+***Learning Objective:***
+
+In this tutorial we look at alphabets and you will learn how to work with nucleotides and amino acids in SeqAn3.
+We guide you through the most important properties of SeqAn's alphabets and show you related concepts.
+After completion, you will be able to use the alphabets inside of STL containers and to implement your own alphabet.
+
+\tutorial_head{Moderate, 60 min, \ref setup, [Concepts](https://en.cppreference.com/w/cpp/language/constraints)}
+
+The links on this page mostly point straight into the API documentation, which you should use as a reference.
+The code examples and exercises are designed to provide some practical experience with our interface
+as well as a code basis for your own program development.
+
+[TOC]
+
+<br/>
+
+---
+
+# Introduction
+
+An alphabet is the set of symbols, of which a biological sequence – or in general a text – is composed.
+SeqAn implements specific and optimised alphabets not only for sequences of RNA, DNA and protein components,
+but also for quality, secondary structure and gap annotation, as well as combinations of the aforementioned.
+
+\assignment{Task}
+Read the section *Detailed Description* of the API reference page for \ref alphabet.
+This is a detailed introduction to the Alphabet module and demonstrates its main advantages.
+\endassignment
+
+<br/>
+
+---
+
+# The nucleotide alphabets
+
+Nucleotides are the components of (Deoxy)Ribonucleic acid (DNA/RNA) and contain one of the nucleobases
+Adenine (A), Cytosine (C), Guanine (G), Thymine (T, only DNA) and Uracil (U, only RNA).
+In SeqAn the alphabets seqan3::dna4 and seqan3::rna4 contain exactly the four respective nucleotides.
+The trailed number in the alphabets' name represents the number of entities the alphabet holds –
+we denote this number as *alphabet size*.
+For instance, the alphabet seqan3::dna5 represents five entities as it contains the additional symbol 'N'
+to refer to an unknown nucleotide.
+
+<br/>
+
+## Creation of alphabet symbols
+Let's look at some example code, which demonstrates how characters of the seqan3::dna4 alphabet are created.
+\snippet alphabet_main.cpp create
+\snippet alphabet_main.cpp closing
+
+We have shown three solutions for assigning variables of alphabet type.
+1. Assignment by character literal, i.e. appending the operator `_dna4` to the respective char symbol. <br/>
+   This is the handiest way, as it can be used temporarily (without creating a variable) as well.
```

Comment:
:thinking: it does create a variable though doesn't it?

Reply:
If you assign it to a variable, then yes; however there is no named object just from the literal. 
Anyway, I should make this statement clearer.